docs/src/pages/migrations.md

```diff
@@ -176,9 +176,15 @@ The following restructuring was done to achieve a more consistent grouping:
 
 #### Rootless images
 
-All Woodpecker images now use a non-privileged user (`woodpecker`) by default.
+The `server` and `cli` images now use a non-privileged user (`woodpecker`) by default.
 If you have volume mounts attached to containers, you might need to update the ownership of these directories from `root` to `woodpecker`.
 
+:::note
+The agent image has to stay rootful by default to be able to mount the `docker.sock` for VM deployments of Woodpecker.
```

Comment:
This wording is a bit confusing. VM or bare-metal or whatever doesn't matter here. The point is, if the agent is used for the `docker` backend (which is the default) it always needs access to the socket.

Reply:
"default" is what the admin deploys. But yes, referring to the backend name makes more sense.